lib/ddtrace/contrib/httparty/module_patch.rb

```diff
@@ -0,0 +1,30 @@
+require 'ddtrace/ext/net'
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/http_propagator'
+require 'ddtrace/contrib/rest_client/ext'
+
+module Datadog
+  module Contrib
+    module HTTParty
+      # HTTParty ModulePatch
+      module ModulePatch
```

Comment:
I would suggest a name change to communicate intent over implementation:

```suggestion
      module Helpers
```

Might make more sense to nest this under `Instrumentation` as well e.g.:

 - `Datadog::Contrib::HTTParty::Instrumentation::Helpers`
 - `Datadog::Contrib::HTTParty::Instrumentation::Request`

Similarly the files would have to be renamed and nested to reflect the namespace change too; 
 - `contrib/httparty/instrumentation/helpers.rb`
 - `contrib/httparty/instrumentation/request.rb`

The names are a bit longer, but I think something like that might better communicate purpose. Open to suggestions for other names too!